### ISSUE TITLE
Ability to pass locale to pivot()

### ISSUE THREAD

**Issue by glebfox** [nicolaskruchten/pivottable#591]
As described [here](https://github.com/nicolaskruchten/pivottable/wiki/Parameters) `pivotUI()` has the `locale ` parameter:

> pivotUI(input [,options [,overwrite [,locale]]])

but `pivot()` doesn't. 

> pivot(input [,options])

and locale hardcoded:

```
defaults = {
...
   localeStrings: locales.en.localeStrings
};
```

It would be very convenient to have the same `locale` parameter for `pivot()` as well as in `pivotUI()`.

Sorry, if I missed any guides how to localize `pivot()`.

Regards,

Gleb.

**Comment by nicolaskruchten** [comment]
You can localize `pivot()` by passing in `localeStrings` with the options object. This is also possible with `pivotUI()` but the extra `locale` parameter to that function is a shortcut.

**Comment by glebfox** [comment]
But it isn't obvious that I have to pass 'localeStrings' to 'rendererOptions' too. Or I missed this point in documentation? If so, then sorry.

**Comment by nicolaskruchten** [comment]
You're right that this is not very clear. I'll think about the implications of adding the `locale` parameter to `pivot()`

**Comment by glebfox** [comment]
Thx. Also I probably found a bug with localization. If I pass a `locale` to `pivotUI()` and my configuration contains `rendererOptions` (for example this [sample](http://nicolas.kruchten.com/pivottable/examples/montreal_2014.html) with localization) then renderer will have no localization. To avoid this I have to pass `localeStrings` to `rendererOptions` by myself. Is it expected behaviour?

**Comment by nicolaskruchten** [comment]
OK @glebfox I think the recent commits to master should fix both of these issues: you can now specify `locale` for both `pivot()` and `pivotUI()` and the options and locale strings are more sensibly merged together, so you should not need to pass `localeStrings` through `rendererOptions` if you are using the top-level `locale` parameter.

If you have a second, can you confirm this on your end and I'll do a proper release and update the documentation?

**Comment by glebfox** [comment]
Hi, I think everything is OK, thx for your fix.

Regards, 

Gleb

**Comment by nicolaskruchten** [comment]
Super! I've just released the change as version 2.6.0 and updated the documentation. Thanks for the bug report :)
